test/verify/check-metrics

```diff
@@ -758,6 +758,28 @@ class TestCurrentMetrics(MachineCase):
         with b.wait_timeout(300):
             b.wait(lambda: float(b.text("#load-avg .pf-l-flex div:first-child").split()[-1].rstrip(',')) < 10)
 
+        # File with CPU temperature does not exist, nothnig is displayed
+        b.wait_not_in_text("#current-metrics-card-cpu", "42 °C")
```

Comment:
This is too specific for testing a negative. This should be more precise, such as `wait_text(..., "")`.